revolv/administrator/views.py

```diff
@@ -31,5 +33,23 @@ class AdministratorEmailView(UserDataMixin, TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super(AdministratorEmailView, self).get_context_data(**kwargs)
-        context['subscribed_user_emails'] = RevolvUserProfile.objects.get_subscribed_to_newsletter()
+
+        user_emails = [r.user.email for r in RevolvUserProfile.objects.get_subscribed_to_newsletter()]
```

Comment:
Doesn't `get_subscribed_to_newsletter` return a `QuerySet`? If so, we could just to `.get_subscribed_to_newsletter().values("user__email")` instead of this list comprehension right?